ext/storage_sources/s3_store/test/test_s3_connection.cpp

```diff
@@ -226,8 +227,8 @@ TestObjectExists(const Aws::S3Crt::ClientConfiguration &config)
     bool exists = false;
     int ret = 1;
 
-    if (!conn.ListBuckets(buckets))
-        return 1;
+    if (ret = conn.ListBuckets(buckets) != 0)
+        return (1);
```

Comment:
If we don't return `ret`, we don't need to assign `ret` to `conn.ListBuckets(buckets)`